BaseFilter/sections/general_extensions.txt

```diff
@@ -80,7 +80,7 @@ flashx.*#%#//scriptlet('prevent-setTimeout', '/RTCPeerConnection[\s\S]*?new Mous
 !
 ! BetterJsPop
 player.hdgay.net,player.subespanolvip.com,3pornstarmovies.com,identi.li,voyeurhit.com,katestube.com,hanime.tv,goldesel.to,fetishshrine.com,homemoviestube.com,fantagazzetta.com,sleazyneasy.com,movie-blog.org,xxxstreams.org,wankoz.com,ikshow.net,xvideosx.com.br,pornicom.com,shesfreaky.com,nosteam.ro,sheshaft.com,dramacity.se,firstsrowsports.tv,pornwhite.com,exvagos1.com,sexgalaxy.net,waybig.com,firstrows.xyz,yes.xxx,porn18sex.com,sextop1.net,hd-world.org,streamingporn.xyz,hotcelebshome.com,imgcloud.pw,queerclick.com,newestxxx.com,rojadiracta.me,imageweb.ws,freepornhq.xxx,sextop.net,firstrowi.com,justpicsplease.com,kisshentai.net,xvideos5.com.br,cam-archive.com,julesjordanvideo.com,ddl-mdh.org,pornomineiro.com,yeswegays.com,myp1p.eu,fullxxxmovies.net,warez-world.org,myp2p.biz,videosdesexoxxx.blog.br,piratecams.com,freepornstreams.org,cricfree.org,mamasijaya.me,mypornstarbook.net,sexhayvc.com,atdhe.to,vrsmash.com,webdesigndev.com,line25.com,stream2watch.eu,javhdmovies.net,redvideos.com.br,handjobhub.com,femdomtube.xxx,porntube4k.net,1kino.in,firstrow1uk.eu,lavoratorio.it,scatkings.com,wiziwig.ru,sportlemon.net,ultradownloads.com.br,javstreams.org,xnxxhamster.net,downloadableporn.org,xxxvideosporno.blog.br,amateurbusters.com,pornclipsxxx.com,pornstreams.org,firstrowsportes.tv,ilivestream.com,porn2dl.net,guyswithiphones.com,streamhunter.net,deep-warez.org,pornogiusto.com,vipbox.biz,phimmup.net,realstreamunited.tv,kissdoujin.com,astucito.com,pussylicking.me,pornotube.it,sexiestpicture.com,fullpornmovies.net,pornbust.net,krank.de,firstrow.top,creamypussyvideos.com,firstrows.net,pirateporn.net,freepornpass.biz,hahasports.net,sportlemons.net,nastyplace.org,p2p4u.tv,xxxphoto.org,lawanswers.com.au,blow-jobs.me,firstsrowsports.eu,ilemi.co,wiziwigs.eu,sportlemons.org,goatd.me,freestreams.eup,pornotubo.it,divxtotal.com,firstsrow.eu,pornilly.com,firstrowi.eu,yougoggle.com,firstrowi.net,pornsextube.tv,firstrow1us.eu,streamsport.eu,adultxpedia.com,xhamster.it,velhas10.com,bibledusexe.com,atdhe.me,annangelxxx.com,globaltravel.com,stopstreams.tv,olpair.com,filecrypt.cc,theteenbay.co,turbobit.net,sankakucomplex.com,pervclips.com,fuckhardporn.com,streamango.com,solidfiles.com,uploaded.net,safelinking.net,openload.co#%#Object.defineProperty(window, 'BetterJsPop', { get: function() { return { init: function() { } }; } });
-oladblock.me,oladblock.xyz,oload.space,oload.live,oload.club,openload.pw,oload.fun,turbobit.cc,fruithosts.net,oload.icu,empressleak.biz,oload.win#%#Object.defineProperty(window, 'BetterJsPop', { get: function() { return { init: function() { } }; } });
+oladblock.xyz,oload.space,oload.live,oload.club,openload.pw,oload.fun,turbobit.cc,fruithosts.net,oload.icu,empressleak.biz,oload.win#%#Object.defineProperty(window, 'BetterJsPop', { get: function() { return { init: function() { } }; } });
```

Comment:
What about `oladblock.xyz`? It does not open on my end.

Reply:
Same.

Reply:
That's why I suggested to remove/convert rules by the rules list. These openload missors are dead and can be removed in one commit instead of 10.

Reply:
So, can we remove it here instead of opening the new PR?

Reply:
> So, can we remove them here instead of opening the new PR?

I think so.

Reply:
I prefer to make bulk changes when removing dead domains, through PRs (see my PRs in this task)